src/eventgrid/azext_eventgrid/_params.py

```diff
@@ -47,17 +47,29 @@
 )
 
 sku_type = CLIArgumentType(
-    help="The Sku name of the resource. The possible values are basic or premium.",
+    help="The Sku name of the resource.",
     arg_type=get_enum_type(['basic', 'premium']),
     options_list=['--sku']
 )
 
 identity_type = CLIArgumentType(
-    help="The identity type of the resource. The possible values are noidentity or systemassigned.",
+    help="The identity type of the resource (e.g., topic or domain).",
```

Comment:
What about allowed types here?

Reply:
The possible values still show in the help text. However, these are redundantly shown hence the removal